vipps-report-api-faq.md

```diff
@@ -123,15 +124,26 @@ The `sincePayout` works in a similar way.
 an amount is always withheld to compensate for high risk, etc. Due to this, each
 payment may not always be connected to a specific payout.
 
+## How can I get access to the API?
+
+The API is available for sale units that already have access to a
+[Vipps API](https://vippsas.github.io/vipps-developer-docs/docs/APIs).
+
+If you do not already have API access, you can not get access to the Report API.
+This is the case if you only have a "Vippsnummer" sale unit, since those
+sale units have no API access.
+
 ## Which API keys give access to the API?
 
 For now: The following API keys give access to the Report API:
 
-* The merchant's own API keys: The same API keys that are used to make payments.
+* The merchant's own API keys: The same API keys that are used to make payments, etc. See
+  [Vipps APIs]([Vipps API](https://vippsas.github.io/vipps-developer-docs/docs/APIs).
```

Comment:
Maybe you want to use the API keys URL instead?
https://vippsas.github.io/vipps-developer-docs/docs/vipps-developers/common-topics/api-keys

Reply:
Good idea!